### DISCUSSION TITLE
How to import Tailwind types now that tailwindcss/tailwind-config doesn't exist?

### DISCUSSION THREAD

**Discussion by grantspilsbury** [tailwindlabs/tailwindcss#13688] [Help]
I am trying to update some old code. I used to import Tailwind types like
`import { TailwindColorConfig, TailwindColorGroup } from 'tailwindcss/tailwind-config'`

But now
`Cannot find module 'tailwindcss/tailwind-config' or its corresponding type declarations.ts  `  

This is the file

```
import resolveConfig from 'tailwindcss/resolveConfig'
import { TailwindColorConfig, TailwindColorGroup } from 'tailwindcss/tailwind-config'
import tailwindConfig from '../../tailwind.config'

export interface ThemeColorConfig extends TailwindColorConfig {
  primary: TailwindColorGroup
}

export const useThemeColors = (): ThemeColorConfig => {
  const twConfig = resolveConfig(tailwindConfig)
  if (!twConfig?.theme?.colors) throw new Error('Missing Tailwind config')
  return twConfig.theme.colors as ThemeColorConfig
}
```

How to import Tailwind types now that tailwindcss/tailwind-config doesn't exist?

**Comment by adam-marshall** [comment]
I have the same issue - just updated from 3.4.1 to 3.4.4.

Cannot see any documentation on this.
